### ISSUE TITLE
Specifying fields to populate on content get does not work

### ISSUE THREAD

**Issue by rubenLindstrom** [flamelink/flamelink#90]
In my case, attempting to fetch an **ad**-entity which has a **company**-field (which is a relative field, pointing to a company-entity). I am only interested in the id and the name of the company. As per the [documentation](https://flamelink.github.io/flamelink/#/content?id=get), my implementation looks like this.

```javascript
require("flamelink/content");
const flamelink = require("flamelink/app");
const admin = require("firebase-admin");

const serviceAccount = require("../../admin.json");
const firebaseApp = admin.initializeApp({
  credential: admin.credential.cert(serviceAccount),
  databaseURL: "https://example.firebaseio.com"
});

const flamelinkApp = flamelink({ firebaseApp, dbType: "cf", isAdminApp: true });
const db = flamelinkApp.content;

exports.getAds = (req, res) => {
  db.get({
    schemaKey: "ads",
    populate: [
      {
        field: "company",
        fields: ["id", "name"]
      }
    ]
  })
    .then(data =>
      res.json(
        data
          ? Object.keys(data).reduce(
              (acc, key) =>
                (acc[key] = {
                  ...data[key],
                  _fl_meta_: undefined
                }),
              {}
            )
          : {}
      )
    )
    .catch(err => {
      return res.status(500).json({ error: err.code });
    });
};
```

Instead of returning the company-entities name and id, as expected, it simply returns the entire object.

And this is my package.json
```json
{
  "name": "functions",
  "description": "Cloud Functions for Firebase",
  "scripts": {
    "serve": "firebase serve --only functions",
    "shell": "firebase functions:shell",
    "start": "npm run shell",
    "deploy": "firebase deploy --only functions",
    "logs": "firebase functions:log"
  },
  "engines": {
    "node": ">=12.0.0"
  },
  "dependencies": {
    "express": "^4.17.1",
    "firebase": "^7.8.2",
    "firebase-admin": "^8.9.2",
    "firebase-functions": "^3.3.0",
    "flamelink": "^1.0.0-alpha.28"
  },
  "devDependencies": {
    "firebase-functions-test": "^0.1.6"
  },
  "private": true,
  "version": "1.0.0",
  "main": "index.js",
  "keywords": [],
  "author": "",
  "license": "ISC"
}
```

**Comment by fabien-lg** [comment]
I have the same issue

**Comment by jperasmus** [comment]
Can you try with version `1.0.0-alpha.30` of the SDK?

**Comment by csabaxyz** [comment]
@jperasmus I'm using `1.0.0-alpha.30` but I have the same issue. Or actually worse than what is described above.
My code looks like this:
```js
  const data = await db.content.get({
    schemaKey: 'videos',
    fields: ['id', 'videoTitle', 'nextVideo'],
    populate: [{
      field: 'nextVideo',
      fields: [ 'id' ]
    }]
  })
```
When I remove the `fields` from the populate, the full object is returned, however, when I add it the object looks like this:
```json
{
  "id": "some-id",
  "videoTitle": "some video title",
  "nextVideo": {
    "isLive": {},
    "order": {},
    "dateExpire": {},
    "artists": {},
    "_fl_meta_": {},
    "videoThumbnail": {},
    "videoTitle": {},
    "id": {}
  }
}
```
Every key gets assigned an empty object, instead of just keeping the `id` as I expect it.

Note, this does seem to work when the `select-relational` field is set to multiple, but where I'm having problems is when it is `multiple:false`.

**Comment by gitdubz** [comment]
@fabien-lg @rubenLindstrom please refer to 
https://github.com/flamelink/flamelink-js-sdk/issues/130

The issue has been resolved in 1.0.0-alpha.31
